src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapStoreScenario.scala

```diff
@@ -0,0 +1,37 @@
+package com.linagora.gatling.imap.scenario
+
+import java.util.Calendar
+
+import com.linagora.gatling.imap.PreDef._
+import com.linagora.gatling.imap.protocol.command.MessageRange._
+import com.linagora.gatling.imap.protocol.command.{MessageRanges, Silent, StoreFlags}
+import io.gatling.core.Predef._
+
+import scala.collection.immutable.Seq
+import scala.concurrent.duration._
+
+object ImapStoreScenario {
+  val numberOfMailInInbox = 10
+  private val appendGracePeriod = 5 milliseconds
+
+  val populateMailbox = exec(imap("append").append("INBOX", Option.empty[Seq[String]], Option.empty[Calendar],
+    """From: expeditor@example.com
+      |To: recipient@example.com
+      |Subject: test subject
+      |
+      |Test content
+      |abcdefghijklmnopqrstuvwxyz
+      |0123456789""".stripMargin).check(ok))
+
+  val populateInbox = repeat(numberOfMailInInbox)(pause(appendGracePeriod).exec(populateMailbox))
+
+  val rangeFlagsUpdates = imap("storeAll").store(MessageRanges(From(1L)), StoreFlags.FlagAdd(Silent.Enable(), "all${loopId}")).check(ok)
+  val singleFlagsUpdate = imap("storeOne").store(MessageRanges(One(1L)), StoreFlags.FlagAdd(Silent.Enable(), "one${loopId}")).check(ok)
```

Comment:
Because:
 - 1. Never done anywhere else
 - 2. I am testing STORE, not APPEND....
 - 3. We don't know much how to do. We need a value each time we send a message, not one for the whole scenario.